<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNew</class>
 <widget class="QWidget" name="AddNew">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>25</number>
   </property>
   <property name="margin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="title">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Add new</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pictures:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="mainPhoto">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="app.qrc">:/icons/image</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Click on image to set as main. Double click to enlarge.</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>true</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="selectionRectVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="addPhoto">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/icons/plus</normaloff>:/icons/plus</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removePhoto">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset resource="app.qrc">
             <normaloff>:/icons/minus</normaloff>:/icons/minus</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Data:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="type"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Flowers:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="flowers"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Flavour:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="flavour"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Flowering time:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="flowering"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Bush size:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="bush"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Frost resistance:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="frost"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QTextBrowser" name="desc">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>12</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>addPhoto</tabstop>
  <tabstop>removePhoto</tabstop>
  <tabstop>name</tabstop>
  <tabstop>type</tabstop>
  <tabstop>flowers</tabstop>
  <tabstop>flavour</tabstop>
  <tabstop>flowering</tabstop>
  <tabstop>bush</tabstop>
  <tabstop>frost</tabstop>
  <tabstop>desc</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
